controller/controller.js

```diff
@@ -18,8 +18,10 @@ exports.getCategories = (req, res, next) => {
 };
 
 exports.getReviews = (req, res, next) => {
-  selectReviews().then((reviews) => {
+  selectReviews(req.query).then((reviews) => {
```

Comment:
I would recommend destructuring your queries here and passing them into the model individually, it will make them easier to interact with in the model.